scripts/watch-demo

```diff
@@ -26,7 +26,12 @@ const watchify = spawn(cmd, [startTsFilePath, '-p', '[tsify]', '-v', '--debug',
 watchify.stdout.pipe(process.stdout);
 watchify.stderr.pipe(process.stderr);
 
-const httpCmd = path.join('node_modules', '.bin', 'http-server');
-const httpServer = spawn(httpCmd, ['-c-1'], { detached: false});
-httpServer.stdout.pipe(process.stdout);
-httpServer.stderr.pipe(process.stderr);
+console.log('Waiting for initial compile...');
+watchify.stderr.on('data', (data) => {
```

Comment:
@Lewuathe  Just curious: it's a bit strange that this is waiting on `stderr` and not `stdout`? any reason for that?

Reply:
@dsmilkov Looks like `watchify` returns the string ("written to...") in `stderr` instead of `stdout`. Actually when I changed to `stdout`, http server was not triggered.

Reply:
Got it. Thanks for double checking!